OpenSim/Analyses/StaticOptimization.cpp

```diff
@@ -553,7 +562,7 @@ int StaticOptimization::begin(const SimTK::State& s )
         sWorkingCopy.setTime(s.getTime());
         sWorkingCopy.setQ(s.getQ());
         sWorkingCopy.setU(s.getU());
-        sWorkingCopy.setZ(s.getZ());
+        // No need to copy Zs to be consistent with record method below 
```

Comment:
I don't understand why Zs are no longer copied, even for enabled muscles?

Reply:
See thread here:
![](https://user-images.githubusercontent.com/4203505/60303249-e7895700-9903-11e9-8558-b657843bc015.png)

Reply:
Thanks @tkuchida . I like your Paint job.